cmd/litestream/mcp/mcp.go

```diff
@@ -0,0 +1,58 @@
+package mcp
+
+import (
+	"context"
+	"fmt"
+	"log/slog"
+	"net/http"
+	"time"
+
+	"github.com/MadAppGang/httplog"
+	"github.com/benbjohnson/litestream/cmd/litestream/mcp/tools"
```

Comment:
Do these functions need to live in a separate package? It seems like they could live in `cmd/litetream`.

Reply:
No, I just wasn't sure how much you wanted in the main folder.  Do you want all the tools and the mcp all in the `cmd/litestream`?  Just want to make sure before I move all them into main.

I've been beaten into submission enough times not to put stuff in main I think it was just a natural reaction.  But I get this is a super small project and probably doesn't matter that much.  I just won't tell the Go deities what we did here... :-).

Reply:
Yeah, go ahead and throw it in `main`. You could even put them all in one `mcp` file. The tools themselves aren't very big.